lib/constants.dart

```diff
@@ -1,5 +1,6 @@
 class AppConstsnts {
```

Comment:
Rename this too. Ofc this doesn't make sense.😹

Reply:
Hmm, looks like you need to merge the latest changes from `upstream`. A PR has been already made, 🙂

Reply:
😅 fixed after some few issues